spec/rubocop/cop/layout/line_length_spec.rb

```diff
@@ -261,6 +261,22 @@ def test_some_other_long_test_description_which_exceeds_length
       RUBY
     end
 
+    context 'and SplitStrings option is enabled' do
+      let(:cop_config) do
+        { 'Max' => 80, 'AllowHeredoc' => true, 'SplitStrings' => true }
+      end
+
+      it 'does not crash' do
+        expect do
+          expect_no_offenses(<<~RUBY)
+            <<~MESSAGE
+              #{'hello' * 1} #{'world' * 2} #{'hello' * 1} #{'world' * 2} #{'hello' * 1} #{'world' * 2}
+            MESSAGE
+          RUBY
+        end.not_to raise_error
+      end
```

Comment:
To make the test reproducible, the heredoc identifier needs to be enclosed in single quotes. Also, `expect do ... end.not_to raise_error` is redundant and should be removed.

```suggestion
      it 'does not register an offense' do
        expect_no_offenses(<<~'RUBY')
          <<~MESSAGE
            #{'hello' * 1} #{'world' * 2} #{'hello' * 1} #{'world' * 2} #{'hello' * 1} #{'world' * 2}
          MESSAGE
        RUBY
      end
```

Reply:
@koic thank you! That made it reproducible, and the test still fails so I adjusted the code to make it pass. Let me know if the changes make sense now.